kafka-streams-quickstart/README.md

```diff
@@ -32,14 +32,20 @@ A Docker Compose file is provided for running all the components.
 Start all containers by running
 
 ```bash
-docker-compose up --build
+docker-compose up -d --build
+```
+
+Due to an [an issue](https://github.com/smallrye/smallrye-reactive-messaging/issues/128) in the SmallRye Reactive Messaging component it might be that the producer application isn’t sending events if it was started before Apache Kafka was up. In this case, simply restart the producer (run from another shell window, keeping the other services running):
```

Comment:
```suggestion
Due to [an issue](https://github.com/smallrye/smallrye-reactive-messaging/issues/128) in the SmallRye Reactive Messaging component it might be that the producer application isn’t sending events if it was started before Apache Kafka was up. In this case, simply restart the producer (run from another shell window, keeping the other services running):
```

Reply:
The SmallRye issue has been resolved some time ago, so actually the entire note can be removed.

Reply:
Oki, but I encountered the problem on my computer and I found this [Quarkus guide about kafak-streams](https://quarkus.io/guides/kafka-streams-guide) that mentions the producer restart.

> You should see log statements from the producer application about messages being sent to the "temperature-values" topic. Due to an [an issue](https://github.com/smallrye/smallrye-reactive-messaging/issues/128) in the SmallRye Reactive Messaging component it might be that the producer application isn’t sending events if it was started before Apache Kafka was up. In this case, simply restart the producer (run from another shell window, keeping the other services running):